### PR TITLE
Things,...

### PR DESCRIPTION
**Pull request by otobrglez**
- [x] Dockerfile for easy docker building and running.
- [x] PM2 for process management
- [x] nodejs "official" docker image
- [x] callback for server starting
- [x] `"use strict";` flag for strict JS rules
- [x] Code formatting
- [x] `processes.json` for process definition
- [x] Updated instructions in `README.md`

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -1,43 +1,46 @@
...
-      },
-      loginCallback: function(resp) {
-      }
-   }
-
-   var itunes = new itc.Connect(user.name, user.pass, options);
-   req.itunes = itunes;
-
-   next();
+app.all('*', function (req, res, next) {
```
Comment (otobrglez) [inline src/server.js:11]: I'm pretty sure that you can do this with [middleware](http://expressjs.com/guide/using-middleware.html) and with `use` function.

File: src/server.js [hunk 2]
```diff
@@ -1,43 +1,46 @@
...
+    req.itunes.request(Report('timed').time(1, 'days').interval('day'), function (error, result) {
+        if (error !== null) {
+            return res.status(500).send({error: 'Something blew up.'});
+        }
 
-      var data = result[0].data[0];
-      res.send(JSON.stringify(data));
-   });
+        var data = result[0].data[0];
+        res.send(JSON.stringify(data));
```
Comment (otobrglez) [inline src/server.js:39]: Usually JSON.stringify is not needed. Just do `res.send(data)` ?

Comment (JanHalozan) [inline src/server.js:39]: Indeed. Fixed.

**Comment by JanHalozan** [comment]
Good stuff man. Thanks.

**Pull request merged**